app/assets/templates/layout/navbar_right.html.slim

```diff
@@ -1,12 +1,12 @@
 ul.nav.navbar-nav.navbar-right
   li.my-account
-    a(ng-click="goToMyAccount()") My Account
+    a(ng-click="goToMyAccount()")= I18n.t 'sprangular.my_account'
```

Comment:
Ah cool, that will work.

Currently we have 2 ways that angular will find templates:

- Templates can be cached inline in the html with a `script` tag, eg: `<script type="text/ng-template" id="<path>"/>`
- Templates can be requested with HTTP GET. That goes through the `railsAssets` interceptor https://github.com/sprangular/sprangular/blob/master/app/assets/javascripts/sprangular/interceptors/railsAssets.coffee#L5 

The `railsAssets` interceptor will need to be updated to inject `current_locale` into the path.